bot/src/bounded-contexts/analytics/domain/services/processors/ShipTypesDataProcessor.ts

```
@@ -0,0 +1,168 @@
+   * Process ship types that were destroyed
+   */
+  private processShipTypesDestroyed(
+    rawData: any[], 
+    config: ChartConfiguration
+  ): ChartData {
+    
+    const shipTypeMap = new Map<string, number>();
+    
+    // Process victim data
+    rawData.forEach(kill => {
+      if (kill.victim && kill.victim.shipTypeName) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)